README.rst

```diff
@@ -0,0 +1,27 @@
+Usage
+=====
+
+In order to create proper plugin repository structure, please execute following steps:
+
+1. `Install cookiecutter <http://cookiecutter.readthedocs.org/en/latest/installation.html>`_
+
+2. Install NodeConductor plugin cookiecutter:
+
+  .. code-block:: bash
+
+    cookiecutter https://github.com/opennode/cookiecutter-nodeconductor-plugin.git
+
+
+You will be prompted to enter values of some variables.
+Note, that in brackets will be suggested default values.
+
+- plugin_name - name of the plugin.
+  **Important! It should start with word NodeConductor** i.e. `NodeConductor SugarCRM`
+- plugin_repo - name of the plugin's repository. i.e. `nodeconductor-sugarcrm`
```

Comment:
it should include github's organisation as well, in case plugin is not hosted under opennode's organisation.